glossary.yml

```diff
@@ -5221,6 +5224,11 @@
     term: የጊት ቅርንጫፍ
     def: >
       የአንድ [Git](#git) [ማከማቻ](#repository) ስሪት ቅጽበታዊ ገጽ እይታ። በርካታ ቅርንጫፎች አንድ ዓይነት ማከማቻ ብዙ ስሪቶችን መያዝ ይችላሉ።
+  uk:
+    term: "Гілка Git"
+    def: >
+      Відгалуження від основної гілки розробки, що дозволяє ізолювати роботу над певними завданнями, 
+      такими як нові функції або виправлення помилок, і уникати конфліктів зі змінами в основній гілці.
```

Comment:
Let's replace "і уникати конфліктів зі змінами в основній гілці" by "незалежно від змін в основній гілці чи інших відгалуженнях"? Since conflicts still may happen when merging - the current translation may be mistakenly interpreted like something helping to avoid conflicts at all.

Reply:
Summarising: 

Відгалуження від основної гілки розробки, що дозволяє ізолювати роботу над певними завданнями, такими як додання нових функцій або виправлення помилок, незалежно від змін в основній гілці чи інших відгалуженнях.

How does this sound?

Reply:
@olexandr-konovalov sounds prefect, I have updated

and I will be more careful with merging in the future